src/procedures/UpdateDividendsTaxWithholdingList.ts

```diff
@@ -32,11 +36,20 @@ export const updateDividendsTaxWithholdingListResolver = (
   );
 };
 
+/**
+ * Procedure that update withholding tax rates for all the investors of the security token
```

Comment:
Procedure that modifies dividend tax withholding percentages for holders of the Security Token